package.json

```diff
@@ -19,8 +19,8 @@
     "@react-navigation/material-top-tabs": "^6.6.10",
     "@react-navigation/native": "^6.1.9",
     "@react-navigation/native-stack": "^6.9.17",
-    "@types/react-native-modern-datepicker": "^1.0.5",
     "@tanstack/react-query": "^5.24.1",
+    "@types/react-native-modern-datepicker": "^1.0.5",
```

Comment:
remove this

Reply:
Updated on [687211b](https://github.com/ParkingSystemWithLPR/Napark-application/pull/24/commits/687211bb293b003be6626ba72bb0c55e065b6fd1)